### DISCUSSION TITLE
Logging to BetterStack

### DISCUSSION THREAD

**Discussion by KaelWD** [coollabsio/coolify#3797] [Q&A]
I want to drain coolify's logs to https://betterstack.com/, I followed the config from https://betterstack.com/docs/logs/fluent-bit/ but it never gets any events. Any ideas what's wrong here?

```
[SERVICE]
    flush  1

[INPUT]
    name tail
    path /var/log/syslog

[OUTPUT]
    name    http
    match   *
    tls     On
    host    in.logs.betterstack.com
    port    443
    uri     /fluentbit
    header  Authorization Bearer [REDACTED]
    header  Content-Type application/msgpack
    format  msgpack
    retry_limit 5
```

**Comment by Arvoid00** [comment]
@KaelWD Same here, did you already have a fix for this?

**Reply by maciejk-code** [reply]
Hey guys, any updates?

**Reply by djsisson** [reply]
if i remember right coolify uses forward (the logs are forwaded to the port below)

so

```
[INPUT]
    Name              forward
    Listen            0.0.0.0
    Port              24224
    Buffer_Chunk_Size 1M
    Buffer_Max_Size   6M
```

Listen might need to be 127.0.0.1

**Reply by Arvoid00** [reply]
@maciejk-code I did it like this. 

FluentBit
```
[SERVICE]
    Parsers_file ./parsers.conf
    Flush 1
    Grace 30

[INPUT]
    Name              forward
    Listen            0.0.0.0
    Port              24224
    Buffer_Chunk_Size 32KB
    Buffer_Max_Size   64KB
    Tag container_logs

[FILTER]
    Name parser
    Match *
    Key_Name log
    Parser jsonparser
    Reserve_Data True

[OUTPUT]
    name    http
    match   *
    tls     On
    host    in.logs.betterstack.com
    port    443
    uri     /fluentbit
    header  Authorization Bearer [REDACTED]
    header  Content-Type application/msgpack
    format  msgpack
    retry_limit 5
```

Parser
```
[PARSER]
    Name        jsonparser
    Format      json
    Time_Key    time
    Time_Format %Y-%m-%dT%H:%M:%S.%L
    Time_Keep   On
```

**Reply by albertorizzi** [reply]
> @maciejk-code I did it like this.

Can you share a guide? Betterstack collets information for all services or only one server?

**Comment by maciejk-code** [comment]
@djsisson @Arvoid00 Hey guys do you also have issues with betterstack? I did have issues since first registration - they said the issue was caused by the source not syncing properly on their backend. Then it worked well for a few days and now the logs aren't loading at all. It's stuck on loading. I'm thinking of moving to some other service. Maybe grafana cloud? I need free tier for now
